plutus-core/plutus-ir/test/TransformSpec.hs

```diff
@@ -133,7 +133,7 @@ letFloatIn =
             ]
   where
     goldenFloatTC pir = rethrow . asIfThrown @(PIR.Error PLC.DefaultUni PLC.DefaultFun ()) $ do
-        let pirFloated = runQuote $ LetFloatIn.floatTerm def pir
+        let pirFloated = runQuote $ LetFloatIn.floatTerm def True pir
```

Comment:
Okay, now I want _one_ test showing that non-relaxed float-in is in fact non-relaxed :)

Reply:
Right! Added a few test cases